docs/docs/ko/server/on-premise/install.md

```diff
@@ -26,7 +26,7 @@ This section outlines the requirements for hosting the Tuist server on your infr
 
 ### Running Docker-virtualized images
 
-We distribute the server as a [Docker](https://www.docker.com/) image via [GitHub’s Container Registry](https://docs.github.com/ko/packages/working-with-a-github-packages-registry/working-with-the-container-registry).
+We distribute the server as a [Docker](https://www.docker.com/) image via [GitHub’s Container Registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry).
```

Comment:
I'll set up a pipeline to automate these through machine translation. Otherwise this can become quite cumbersome to update once they are in Korean.